pallets/token-manager/src/lib.rs

```diff
@@ -238,6 +238,8 @@ pub mod pallet {
         LowerSchedulePeriodUpdated {
             new_period: BlockNumberFor<T>,
         },
+        LoweringEnabled,
+        LoweringDisabled,
```

Comment:
One event for settings is better to avoid bloating the pallet codebase.

Reply:
I personally prefer having specific events that describe exactly what is happening instead of a generic one with different argument. Again happy to hear what others think.